src/helper/FormattedDate.ts

```diff
@@ -0,0 +1,24 @@
+export class myFormattedDate {
+  static convertFormattedDate = () => {
+    const date = new Date();
+    const day = date.getDate();
+    const monthIndex = date.getMonth();
+    const year = date.getFullYear();
+    const minutes = date.getMinutes();
+    const hours = date.getHours();
+    const seconds = date.getSeconds();
+    const myFormattedDate =
```

Comment:
Why this is not defined as a templated string since the string concatenation costs more than string substitution via templating.